pkgs/development/libraries/science/math/costa/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, mpi
+, scalapack
+}:
+
+stdenv.mkDerivation rec {
+  pname = "COSTA";
+  version = "2.2.2";
+
+  src = fetchFromGitHub {
+    owner = "eth-cscs";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-jiAyZXC7wiuEnOLsQFFLxhN3AsGXN09q/gHC2Hrb2gg=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ scalapack ];
+
+  propagatedBuildInputs = [ mpi ];
+
+  cmakeFlags = [
+    "-DCOSTA_SCALAPACK=CUSTOM"
+    "-DSCALAPACK_ROOT=${scalapack}"
+    # Required due to broken CMake files
+    #"-DCMAKE_INSTALL_LIBDIR=lib"
+    #"-DCMAKE_INSTALL_INCLUDEDIR=include"
```

Comment:
```suggestion
```

Not required?

Reply:
Ha, copy and paste leftover from SPLA. :grimacing: